api/crudFunctions/weeklyPicks.tsx

```diff
@@ -0,0 +1,36 @@
+//globally accessible env variable
+declare global {
+    namespace NodeJS {
+        export interface ProcessEnv {
+            [key: string]: string | undefined;
+            PROJECT_ID: string;
+            API_KEY: string;
+        }
+    }
+}
+
+export async function getWeeklyPicks() {
+    try {
+        await fetch(`https://cloud.appwrite.io/v1/databases/${process.env.DATABASE_ID}/collections/${process.env.WEEKLY_PICKS_COLLECTION_ID}/documents`, {
```

Comment:
Please look at all the headers that are needed based on the Appwrite docs. I don't believe you need most of these or any of them.